azavea/commonhealth-data-map
# Show last update time; date selector shows today.

## Overview

Pulls in changes from the EAC repository:
- Add popup to Cases tab.
- Add footer with sources and last updated date.
- Disable country select and date select if not used.
- Show latest date in date selector.
- Remove capacity tab attribution control - sources are now in footer.
- Maintain date selection when crossing tabs.

## Testing Instructions

In both EAC and TCP dashboard:
- Test that date selector shows todays date, sliding dates works 
- Check last update time and sources show in footer for each tab
- Check that date selection and region selection don't show for capacity tab
- Check that popups work for cases tab

Fixes #35
